packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -912,12 +912,14 @@ class DefaultMaterialLocalizations implements MaterialLocalizations {
 
   @override
   String licensesPackageDetailText(int licenseCount) {
-    assert (licenseCount >= 1);
+    assert (licenseCount >= 0);
     switch (licenseCount) {
+      case 0:
+        return 'No licenses.';
       case 1:
-        return '1 license';
+        return '1 license.';
```

Comment:
Assuming that the additional punctuation makes sense; the translations don't include a period.